src/api/Gemfile

```diff
@@ -118,6 +118,8 @@ gem 'rexml'
 
 gem 'base64', '0.2.0', require: false
```

Comment:
This one we are doing for the same reason...

Reply:
added, thanks